releng/tools/org.openjdk.jmc.util.releasenotes/notes/8.2/notes.xml

```diff
@@ -0,0 +1,179 @@
+				trace data from selections in the JFR UI as JSON on a user defined
+				port. This allows for programmatic control of the visualization
+				directly in the browser. Tools like observablehq.com can be used to
+				invent new visualizations, or to alter the visualization. To get
+				started, simply go to the Flight Recorder preferences in JMC, and
+				select the Websocket port to use (0 to disable). A set of example
+				visualizations are available here:
+				https://observablehq.com/collection/@cimi/java-mission-control.
+			</description>
+			<image>jfr/websocket.png</image>
+		</note>
+	</component>
+	<!-- ******************************* Agent ******************************* -->
+	<!-- <component> <name>JMC Agent Plug-in</name> <note version="8.1.0"> <title>New 
+		JMC Agent Plug-in</title> <description> There is now a new agent plug-in 
+		available for JMC, which allows configuring where to emit flight recording 
+		events in an already running process. </description> <image>agent/agent.png</image> 
+		</note> </component> -->
+	<!-- ******************************* Bugfixes ****************************************** -->
+	<bugfixes>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>7403</bugid>
+			<synopsis>JFR parser struct types hashcode problem</synopsis>
+			<description>
+				Some JFR parser struct types were using lazily
+				initialized attributes which happen to be a part of hashCode/equals
+				computations.
+			</description>
+		</bugfix>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>7532</bugid>
+			<synopsis>
+				Delays in rendering of JMX console graphs
+			</synopsis>
+			<description>
+				Sometimes the updates of the JMX console graphs would be
+				severely delayed on MacOS. This is now fixed.
+			</description>
+		</bugfix>
+	</bugfixes>
+	<!-- ******************************* Known issues ****************************************** -->
+	<knownissues>
+		<issue>
+			<area>General</area>
+			<bugid>4270</bugid>
+			<synopsis>Hibernation and time</synopsis>
+			<description>
+				After the bugfix of
+				https://bugs.openjdk.java.net/browse/JDK-6523160 in JDK 8, the
+				RuntimeMXBean#getUptime() attribute was re-implemented to mean
+				"Elapsed time of JVM process", whilst it previously was implemented
+				as time since start of the JVM process. The uptime attribute is used
+				by JMC, together with RuntimeMXBean#getStartTime(), to estimate the
+				actual server time. This means that time stamps, as well as
+				remaining time for a flight recording, can be wrong for processes on
+				machines that have been hibernated.
+			</description>
+		</issue>
+		<issue>
+			<area>JFR</area>
+			<bugid>7071</bugid>
+			<synopsis>JMC can't attach to jlinked JVMs</synopsis>
+			<description>
+				This one is still under investigation, but it seems JMC
+				can't attach to certain jlinked images.
+			</description>
+		</issue>
+		<issue>
+			<area>JFR</area>
+			<bugid>7068</bugid>
```

Comment:
This bug was resolved after an Eclipse update: https://bugs.openjdk.java.net/browse/JMC-7068